data/plugins.d/50-ffmemless.ini

```diff
@@ -44,7 +49,7 @@ supported_effects = touch;touch_weak;touch_strong;release;release_weak;release_s
 #
 # -Type specific parameters for periodic effects
 #
-# _WAVEFORM =	[sine|triangle|square], defaults to sine
+# _WAVEFORM =	[sine|triangle|square|custom], defaults to sine
```

Comment:
Thanks for adding this. 
You'd also want to specify that if _WAVEFORM is custom then _CUSTOM can be set to 1..5 usually, depending on the driver.